apps/redi-talent-pool/src/components/organisms/job-listing-card/ExpiredJobListingCard.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  TpJobListingStatus,
+  useTpJobListingPatchMutation,
+} from '@talent-connect/data-access'
+import { Button } from '@talent-connect/shared-atomic-design-components'
+import { useCallback } from 'react'
+import { Card } from 'react-bulma-components'
+import { queryClient } from '../../../services/api/api'
+import './ExpiredJobListingCard.scss'
+import { JobListingCard, JobListingCardProps } from './JobListingCard'
+
+interface ExpiredJobListingCardProps extends JobListingCardProps {
+  onClick?: (e: React.MouseEvent) => void
+}
+
+export function ExpiredJobListingCard({
+  onClick,
+  ...jobListingCardProps
+}: ExpiredJobListingCardProps) {
+  const updateMutation = useTpJobListingPatchMutation()
+
+  const onReactivateClick = useCallback(() => {
+    updateMutation.mutate(
+      {
+        input: {
+          id: jobListingCardProps.jobListing.id,
+          status: TpJobListingStatus.Active,
+          expiresAt: new Date().setMonth(new Date().getMonth() + 6),
+        },
```

Comment:
We should not (and I assume we cannot) change the `createdAt` date as it's an auto-generated field by Salesforce. The only way would be re-creating the job listing and delete the expired one. I believe the right way of doing this is sorting job listing cards by `udpatedAt` instead of `createdAt`. That field is the right field to keep when the last changes were made to a job listing.

Reply:
Yes, you are right. We want to see the most up-to-date job listings at the top of the search results. In this case, we should sort them by `udpatedAt` and display the `udpatedAt` date as the posting date (or last updated date) in the job listing.